lib/data/seeds/commitments.csv

```diff
@@ -17,6 +17,6 @@ Non-governmental organization ,Spain,Yes,55.37911,-110.308466,1000000,0,2020,202
-Non-governmental organization ,Australia,Yes,55.37911,-110.308466,0,0,2015,2021,less than 25 years,Committed only,2021,Kimbe Bay Hope Spot,Local communities,,target 1,Self-reported,Other,To extend the planned lifespan of an existing area,"The Kimbe Bay battles the negative effects of local human population growth, the expansion of offshore fishing efforts, seabed mining, pollution and warming water temperatures. Hope Spot Champions Cecilie Benjamin and Gemma Galbraith and the students of Marine Environment Education Program (MEEP) are leading the cause to improve enforcement of the areas LMMAs (Locally Managed Marine Areas) and MPAs and to expand their program to continue educating Kimbe Bays youth about reef conservation.",https://storymaps.arcgis.com/stories/2b3e6d9f88614ee593919e2a19e90204,live
-Non-governmental organization ,Uruguay,Yes,55.37911,-110.308466,0,300,2020,2021,less than 25 years,Committed only,2021,Whale and Dolphin Sanctuary of Uruguay Hope Spot,Private ,,target 1,Not Applicable,Other,To collaborate with the managers of neighbouring area to improve conservation in the landscape/seascape,"The Whale and Dolphin Sanctuary of Uruguay has been declared a Hope Spot by international nonprofit Mission Blue in recognition of its value as a safe haven for cetaceans, to establish an enforceable policy to thoroughly manage and protect the sanctuary and to educate the public about the negative effects of ""ocean noise"" on marine life created by oil drilling, shipping and seismic testing.",https://mission-blue.org/2018/10/the-whale-and-dolphin-sanctuary-of-uruguay-declared-a-hope-spot-in-support-of-cetacean-conservation/,live
+Non-governmental organization ,Australia,Yes,55.37911,-110.308466,0,0,2014,2021,less than 25 years,Committed,2021,Salisbury Island Hope Spot,Private ,,target 1,Self-reported,Recreation,To extend the planned lifespan of an existing area,"Remote ecosystem through observation and promote virtual tourism here rather than physical tourism to minimize the human impact on this sensitive location. Conservationists working for Finding Salisbury, Inc. are seeking official government protection to maintain the area's stable condition with a comprehensive no-take state marine reserve and a ""virtual tourism"" program to allow the public virtual access while keeping the area wild. ",https://storymaps.arcgis.com/stories/556d12e023ce4fe68e2d6e44be9cbf42,live
```

Comment:
Guessing the extra spaces around some of the values in this CSV aren't causing problems, since it was like that already?